bot/pydici_bot.py

```diff
@@ -16,6 +16,8 @@
 import random
 import pytz
 
+from staffing.utils import is_outside_business_hours
```

Comment:
This import cannot work so early in the file before env has been set. It fails with ModuleNotFoundError: No module named 'staffing'.
It should be moved around line 49 with others pydici imports.